basics/uri.adoc

```diff
@@ -102,7 +102,7 @@ uEntities *MAY* assume both the Service and Application roles.
 A UUri's `ue_id` property value determines the type and instance of the service being referred to:
 
 * The value's _least_ significant 16 bits contain the _service ID_, representing the service interface type.
-* The value's _most_ significant 16 bits contain the _service instance ID_.
+* The value's _most_ significant 16 bits contain the _service instance ID_ where a value of `0x0000` refers to _any_ (or don't care). 
```

Comment:
IIRC you chatted with Greg about this on Friday/Saturday and he had a different idea which you thumbs-upped.

Reply:
Thanks Pete, I'll push an update with justification shortly